src/main/scala/com/bizo/hive/sparkplug/emr/Emr.scala

```diff
@@ -19,7 +19,8 @@ class Emr private(credentials: Option[AWSCredentials]) {
 
   def this(knownCredentials: AWSCredentials) = this(Some(knownCredentials))
 
-  def run(flow: JobFlow)(implicit config: ClusterConfig): String = {
+  def run(flow: JobFlow, configureRequest: RunJobFlowRequest => RunJobFlowRequest = identity)
+    (implicit config: ClusterConfig): String = {
```

Comment:
Good point but it is backwards compatible due to the default value (which basically does nothing)